src/annotate_variants/tests/test_validation.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from ..tests.mocks import MockResponse
+from ..validation import validate_sys_argv, validate_api_response, validate_path
+
+
+def test_validate_sys_argv():
```

Comment:
Glad to see tests for all of these. I think similar to comments on test_ensembl.py I would like to see separate of each individual test case, using pytest parameterize, also separating out expected successes from expected failures. As these tests are currently setup they require that all previous cases exist successfully for later cases to be tested and status observed. Would be hard to see the blast radius of a bug with this setup

Reply:
Thank you for this comment in particular. I've only recently come across pytest parameterize and am still gaining mastery of it; running tests as I did is an older pattern that got me to the endpoint more quickly and reliably. I'll make a point of practicing pytest parameterize more so that it's more comfortable to reach for first.